tokio-trace/src/span.rs

```diff
@@ -117,23 +252,69 @@
 //! given ID, then no more handles to the span with that ID exist. The
```

Comment:
There's a typo above:

> If there is only a single handle with the capacity to exit a span, dropping that handle **"close"** the span, since the capacity to enter it no longer exists.

Should say "closes"